.github/workflows/ci.yml

```diff
@@ -53,6 +53,14 @@ jobs:
           name: test-result
           path: /home/runner/work/**/build/reports
           retention-days: 2
+      - name: Publish Test Report
+        uses: mikepenz/action-junit-report@v3.7.6
+        if: success() || failure()
+        with:
+          check_name: "test-reports-${{ matrix.alfresco_version }}"
+          report_paths: '**/build/test-results/**/TEST-*.xml'
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Maybe try `detailed_summary` https://github.com/mikepenz/action-junit-report?

Reply:
this adds the following when all tests succeed: 
![image](https://github.com/xenit-eu/alfred-api/assets/121961965/956cb9b0-4f83-4ecd-bf06-23308dba75a8)
This adds the following when test(s) fail:
![image](https://github.com/xenit-eu/alfred-api/assets/121961965/14258a38-6154-4b4f-93fb-a625a87999cf)

Reply:
comparing this to what we see when opening the "test_results":
![image](https://github.com/xenit-eu/alfred-api/assets/121961965/be35c86b-5878-45ce-8636-fc36a85dc1d8)
i think we can let it out

Reply:
I'm ok with both options. Thanks for trying it out.